### ISSUE TITLE
field report

### ISSUE THREAD

**Issue by nickray** [solokeys/solo-desktop#2]
Win10, 1809
- Solo Hacker 2.5.1, wants to update, goes to Solo Bootloader 2.2.2. Error: `ReferenceError: CTAP1_ERR_INVALID_COMMAND is not defined`
- Solo 2.5.2, manually to Solo Bootloader 2.5.2: Same error

App crashes sometimes (not sure how to debug reason on Windows 😁)

Competition gets listed with interesting serial:
![image](https://user-images.githubusercontent.com/1092423/67807506-3521f600-fa95-11e9-8bf9-5f745e0ab9b3.png)

**Comment by My1** [comment]
using 8.1 here, looks fine with my hackersolo and Yubi 4 Nano (post ROCA)
![image](https://user-images.githubusercontent.com/6696524/67853508-fcbefe00-fb0e-11e9-8415-0f65031e4ac3.png)

**Comment by aseigler** [comment]
That fixed the CTAP1_ERR_INVALID_COMMAND for me on 1903.

**Comment by conorpp** [comment]
Can you run it from cmd prompt and past the output when it hangs?

**Comment by aseigler** [comment]
Running in production
list
Error getting CTAP2 info. 11
reply to  list
list

**Comment by conorpp** [comment]
@aseigler what FIDO devices do you have plugged in?

**Comment by aseigler** [comment]
> @aseigler what FIDO devices do you have plugged in?

Good call, by trial and error the problem device is Feitian BioPASSFIDO2.  Both K27 and K33 cause the command line error to be logged, the K27 causes a hang about 50% of the time, if I unplug that device it unblocks.

![image](https://user-images.githubusercontent.com/6605560/67952056-0efa7400-fbc3-11e9-9be0-69545c9f6951.png)
